### PR TITLE
Update docs to Discussions

### PR DESCRIPTION
**Pull request by Airblader**
closes #4482

### CODE REVIEW

File: docs/i3-pod2html [hunk 1]
```diff
@@ -83,7 +83,7 @@ $parser->html_header_after_title(
 			<ul id="nav">
 				<li><a style="border-bottom: 2px solid #fff" href="/docs">Docs</a></li>
 				<li><a href="/screenshots">Screens</a></li>
-				<li><a href="https://www.reddit.com/r/i3wm/">FAQ</a></li>
+				<li><a href="https://www.github.com/i3/i3/discussions">Get Help</a></li>
```
Comment (Airblader) [inline docs/i3-pod2html:86]: It's not really an "FAQ", nor has it ever been one. It's a Q&A. But I think "Get Help" is a bit more understandable as non-native speakers might not be familiar with the abbreviation.

File: release.sh [hunk 2]
```diff
@@ -227,4 +227,4 @@ echo ""
 echo "Announce on:"
 echo "  twitter"
 echo "  #i3 topic"
-echo "  reddit /r/i3wm (link post to changelog)"
+echo "  GitHub Discussions → Announcements"
```
Comment (orestisfl) [inline release.sh:231]: @stapelberg are you planning to  keep making the posts to reddit?

**Review by stapelberg — APPROVED** [review]

**Review by orestisfl — APPROVED** [review]

File: release.sh (hunk 2 above)
Comment (stapelberg) [reply release.sh:231]: Probably good to post in both places, at least for now

Comment (Airblader) [reply release.sh:231]: I've added it back.

**Pull request merged**